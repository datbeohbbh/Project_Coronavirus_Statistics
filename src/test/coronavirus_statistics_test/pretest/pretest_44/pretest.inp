50 100
q 1 7 191
? 6
q 1 43 490
? 6
? 4 2
q 1 19 189
? 4 2
? 4 3
? 4 2
? 4 2
q 1 27 391
q 1 28 112
q 1 40 8
? 4 3
q 1 8 332
q 3 43 3
q 1 33 22
q 2 19 145
q 1 5 494
q 1 37 94
q 1 1 219
q 2 33 4
? 4 5
? 6
q 1 22 440
q 3 19 25
q 1 7 182
q 2 5 436
? 7
q 1 45 234
q 1 34 104
q 1 33 210
q 3 1 109
? 7
? 4 3
q 1 38 250
? 4 15
? 4 8
? 4 6
? 4 15
? 4 5
q 2 22 10
q 1 32 279
? 6
q 1 15 14
q 3 38 186
q 3 7 33
? 4 3
q 2 38 30
q 3 7 271
? 4 6
? 4 17
? 6
q 2 27 137
q 1 10 133
q 1 37 18
q 3 32 210
q 2 8 234
q 2 8 95
q 1 19 436
q 1 44 242
q 3 38 23
? 6
q 3 27 42
q 2 7 11
q 2 1 6
? 4 7
q 3 8 2
q 1 12 449
q 3 28 104
q 1 26 204
q 1 39 349
? 4 10
? 4 2
q 2 33 172
q 1 18 142
? 4 14
q 2 40 3
? 4 4
q 2 32 27
? 7
? 4 13
? 6
? 4 16
? 6
? 7
? 7
? 7
? 4 2
? 6
? 6
? 4 7
? 7
? 4 19
? 7
? 7
? 6
? 4 15
? 6
? 4 18