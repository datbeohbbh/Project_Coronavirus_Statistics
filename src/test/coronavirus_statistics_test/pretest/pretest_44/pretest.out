Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 2 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 490
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 191
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 490
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 191
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 191
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 189
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 112
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 19
Quantity: 145 people

Query ? 7 :
The city with the highest death is city with the index 19
Quantity: 25 people

Query ? 7 :
The city with the highest death is city with the index 1
Quantity: 109 people

Query ? 4 3 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 58
-> Has recovered : 436
-> Was dead : 0

Query ? 4 15 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 487
-> Has recovered : 0
-> Was dead : 3

Query ? 4 8 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 228
-> Has recovered : 4
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 110
-> Has recovered : 0
-> Was dead : 109

Query ? 4 15 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 487
-> Has recovered : 0
-> Was dead : 3

Query ? 4 5 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 104
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 4 3 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 19
-> Has recovered : 145
-> Was dead : 25

Query ? 4 6 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 69
-> Has recovered : 0
-> Was dead : 304

Query ? 4 17 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 487
-> Has recovered : 0
-> Was dead : 3

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 4 7 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 69
-> Has recovered : 0
-> Was dead : 210

Query ? 4 10 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 104
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 142
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 11
-> Has recovered : 30
-> Was dead : 209

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 304 people

Query ? 4 13 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 4 16 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 212
-> Has recovered : 137
-> Was dead : 42

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 304 people

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 304 people

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 304 people

Query ? 4 2 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 5
-> Has recovered : 3
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 4 7 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 56
-> Has recovered : 176
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 304 people

Query ? 4 19 :
The answer is the city with the index 39 
- Information of the city 39 -
-> Number of current patients : 349
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 304 people

Query ? 7 :
The city with the highest death is city with the index 7
Quantity: 304 people

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 4 15 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 204
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 436 people

Query ? 4 18 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 242
-> Has recovered : 0
-> Was dead : 0

